ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```
@@ -53,6 +53,47 @@ tests:
   container:
     clone: true
     from: golangci-lint
+- as: check-commits-count
+  commands: ci/prow/check-commits-count
+  container:
+    clone: true
+    from: src
+  optional: true
+- as: e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 40m0s
+    version: "4.11"
+  steps:
+    test:
+    - as: e2e-incluster-build
```

```suggestion
    - as: e2e-incluster-build-kernel-module
```
Also, I am not sure `steps` is the correct field to use here. Those 2 jobs should run independently.

---

Are different `steps` run in different containers? Are they depending on each other?